fstest/testserver/init.d/TestSeafile

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+# environment variables passed on docker-compose
+export MYSQL_ROOT_PASSWORD=pixenij4zacoguq0kopamid6
+export SEAFILE_ADMIN_EMAIL=seafile@rclone.org
+export SEAFILE_ADMIN_PASSWORD=pixenij4zacoguq0kopamid6
+export SEAFILE_PORT=8088
+export SEAFILE_TEST_DATA=/tmp/seafile-test-data
+
+# local variables
+NAME=seafile
+SEAFILE_IP=127.0.0.1
```

Comment:
This is binding to the world I noticed

```
tcp6       0      0 :::8088                 :::*                    LISTEN      62984/docker-proxy  
```

There are rather a lot of things going on on the integration server and I try to stop the docker stuff binding to IPs owned by the host. It shouldn't be accessible from outside the server. 

We could use the `docker_ip` function from `docker.bash` and just turn off the external port forwarding?

At minimum I'd like it only to bind to 127.0.0.1 not the external IPs.

Oh and while experimenting with this I discovered what you meant

```
$ rclone lsd seafile:
           0 2020-04-30 10:40:33        -1 TestSeafile
          12 2020-04-30 10:40:33        -1 TestSeafile
          30 2020-04-30 10:40:33        -1 TestSeafile
           0 2020-04-30 10:40:33        -1 TestSeafile
          40 2020-04-30 10:40:33        -1 TestSeafile
```



I note that 10.5.0.1 seems to work!

Reply:
Doing routing back to a container like you do only works on linux machines.
On Windows and Mac the containers are behind a NAT so you can't connect back to the container IP directly.
I couldn't run the tests locally with the default configuration.

I can bind the ports to 127.0.0.1 so that would work everywhere.
But if you prefer I will change the configuration to use routing and I'll setup and run the tests manually on my machine.

Reply:
Bind it to 127.0.0.1 is fine if that works for you.

I didn't know that about the direct routing only working on linux machines, maybe I should reconsider how those tests work...